modules/ROOT/pages/to-request-vpc-connectivity.adoc

```diff
@@ -22,13 +22,13 @@ To add additional static routes to Anypoint VPCs that were connected after Novem
 |<<request_vpc_connectivity,Request VPC Connectivity to Your Network>>
 |===
 
-[request_vpc_connectivity]
+[[request_vpc_connectivity]]
 == Request VPC Connectivity
 
 If you need to connect your Anypoint VPC to your internal network using VPN tunnel, peering, or direct connect, you must:
 
 . Download link:{attachmentsdir}/VPC-form-v9.3-template.xlsx[the Anypoint VPC discovery form], which requires at least MS Excel 2007 with macros enabled).
-. Enter the data to communicate the necessary details required for your connectivity method. If you need support for multiple Amazon regions, submit one copy of the discovery form for each region in which you need support.
+. Enter the data to communicate the necessary details required for your connectivity method. If you need support for multiple regions, submit one copy of the discovery form for each region in which you need support.
```

Comment:
Could we please add a note here? Something like this...
NOTE: When updating an existing VPN, the VPC discovery form is only required if the VPN was created prior to November 2018. For all VPNs created after this date, please follow the self-service steps.